era5cli/fetch.py

```diff
@@ -287,16 +288,16 @@ def _check_variable(self, variable):
 
     def _check_area(self):
         """Confirm that area parameters are correct."""
-        (ymax, xmin, ymin, xmax) = self.area
-        if not (-90 <= ymax <= 90
-                and -90 <= ymin <= 90
-                and -180 <= xmin <= 180
-                and -180 <= xmax <= 180
-                and ymax > ymin
-                and xmax != xmin
+        (lat_max, lon_min, lat_min, lon_max) = self.area
+        if not (-90 <= lat_max <= 90
+                and -90 <= lat_min <= 90
+                and -180 <= lon_min <= 180
+                and -180 <= lon_max <= 180
+                and lat_max > lat_min
+                and lon_max != lon_min
                 ):
             raise ValueError(
-                "Provide coordinates as ymax xmin ymin xmax. "
+                "Provide coordinates as lat_max lon_min lat_min lon_max. "
                 "x must be in range -180,+180 and y must be in range -90,+90."
```

Comment:
```suggestion
                "`lat_min`/`lat_max` must be in range -180,+180 and `lon_min`/`lon_max` must be in range -90,+90."
```

Reply:
Yikes! Thanks much for the eagle eyes here!